main/main.c

```diff
@@ -611,6 +611,7 @@ PHP_INI_BEGIN()
 #ifdef PHP_WIN32
 	STD_PHP_INI_BOOLEAN("windows.show_crt_warning",		"0",		PHP_INI_ALL,		OnUpdateBool,			windows_show_crt_warning,			php_core_globals,	core_globals)
 #endif
+	STD_PHP_INI_ENTRY("syslog.highbit",			"0",		 PHP_INI_SYSTEM,	OnUpdateBool,		syslog_highbit,		php_core_globals, 		core_globals)
```

Comment:
> I don't think there is any disagreement about escaping ASCII control characters (which is what the RFC 5424 quote is about and also what the discussion in bug #74860 is about). My gripe is with the high code units only, which don't fall under that clause.

@nikic If `syslog.ascii_only = false` is set, high code units are left unaltered.  Is this not the behavior you're asking for?